class/defaults.yml

```
@@ -1,3 +1,64 @@
 parameters:
   fluentbit:
     namespace: syn-fluentbit
+    configMapName: syn-fluentbit-config
```

The config map is namespaced. Prefexing it with `syn-` is redundant.

```suggestion
    configMapName: fluentbit-config
```

---

I added the `syn-` prefix to make it clearer which configmap contains the Project Syn-managed configuration, since the helm chart also generates a configmap, which is named `fluent-bit` with the current implementation.